### PR TITLE
chore(api-client-app): temp fix icon

### PR DESCRIPTION
**Pull request by marclave**

### CODE REVIEW

File: packages/api-client-app/src/main/index.ts [hunk 1]
```diff
@@ -14,7 +16,7 @@ function createWindow(): void {
     titleBarStyle: 'hidden',
     trafficLightPosition: { x: 11, y: 12 },
     autoHideMenuBar: true,
-    ...(process.platform === 'linux' ? { icon } : {}),
+    icon: iconPath,
```
Comment (hanspagel) [inline packages/api-client-app/src/main/index.ts:19]: If that’s not set, it’s using the app icon. Let’s not change it, until we need to. :v:

File: packages/api-client-app/electron-builder.yml
Comment (hanspagel) [inline packages/api-client-app/electron-builder.yml:1]: That file won’t be used with toDesktop, you can ignore it.

File: packages/api-client-app/build/icon.png
Comment (hanspagel) [inline packages/api-client-app/build/icon.png:1]: Ohhh, where is the Scalar icon going? 🥲 I love the ASCII graphic, but I think the Scalar logo is the better app icon.

File: packages/api-client-app/src/main/index.ts [hunk 2]
```diff
@@ -2,7 +2,9 @@ import { electronApp, is, optimizer } from '@electron-toolkit/utils'
 import { BrowserWindow, app, ipcMain, shell } from 'electron'
 import { join } from 'path'
 
-import icon from '../../build/icon.png?asset'
+const icon = join(process.env.PUBLIC || app.getAppPath(), 'build', 'icon.png')
+
+const iconPath = is.dev ? join(__dirname, '../../build/icon.png') : icon
```
Comment (hanspagel) [inline packages/api-client-app/src/main/index.ts:7]: What’s that doing/changing? 🤔

File: packages/api-client-app/build/icon.png
Comment (cameronrohani) [reply packages/api-client-app/build/icon.png:1]: You could be 100% right, any way I can test multiple logos/icons for this ya'll?

**Comment by marclave** [comment]
closing cause idk what i was cooking here 😅

**Pull request closed without merging**